README.md

```diff
@@ -44,6 +44,9 @@ sudo dnf install pesign yum-utils openssl wget numactl-devel
 sudo dnf builddep kernel-${UNAME%.*}
 sudo dnf debuginfo-install kernel-${UNAME%.*}
 
+# required, on ppc64le for building livepatch modules
```

Comment:
The "for building livepatch modules" is already implicit here.  Also the comma kind of makes it sound like it's always required.  So maybe change it to something like "required on ppc64le"?